src/main/scala/org/hammerlab/guacamole/likelihood/Likelihood.scala

```diff
@@ -157,7 +157,7 @@ object Likelihood {
             case (allele, alleleFraction) =>
               alleleElementProbabilities(alleleToIndex(allele), ::) * alleleFraction
           }
-          sum( log( sum(alleleRows) )) + math.log(prior(genotype)) * depth
+          sum( log( sum(alleleRows) + 1e-10 )) + math.log(prior(genotype))
```

Comment:
OK, different question here: it seems to me that the inner `sum` here is being applied to probabilities, not in log-space, which I don't understand. Is that a bug or am I missing something?